### PR TITLE
VZ-7432 PSR first pass at psrctl CLI tool

### PR DESCRIPTION
**Pull request by pfmackin**
Add psrctl explain do describe available scenarios
Add psrctl start to run the example scenario
Add code to extract embedded manifests file to temp dir

### CODE REVIEW

File: tools/psr/psrctl/cmd/explain/explain.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	"github.com/verrazzano/verrazzano/tools/psr/psrctl/pkg/embedded"
+	"github.com/verrazzano/verrazzano/tools/psr/psrctl/pkg/scenario"
+	cmdhelpers "github.com/verrazzano/verrazzano/tools/vz/cmd/helpers"
+	"github.com/verrazzano/verrazzano/tools/vz/pkg/helpers"
+)
+
+const (
+	CommandName = "explain"
+	helpShort   = "Explain a PSR test scenario"
+	helpLong    = `The command 'explain' explains a available scenario that can be started`
```
Comment (mbenmclean) [inline tools/psr/psrctl/cmd/explain/explain.go:18]: an available

File: tools/psr/psrctl/cmd/explain/explain.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		return explainCmdExplain(cmd, vzHelper)
+	}
+	cmd.Example = helpExample
+
+	return cmd
+}
+
+// explainCmdExplain - explain the "psrctl explain" command
+func explainCmdExplain(cmd *cobra.Command, vzHelper helpers.VZHelper) error {
+	fmt.Println("Listing available scenarios ...")
```
Comment (mbenmclean) [inline tools/psr/psrctl/cmd/explain/explain.go:35]: does this command list the help for one scenario or explain all scenarios. This makes it sound like the latter, but the command help above makes it sound like the former. Please clarify the help if this just lists all the scenario. Maybe change to List scenarios?

Comment (mbenmclean) [reply tools/psr/psrctl/cmd/explain/explain.go:35]: Or perhaps have it take an argument that's the scenario name and list only the scenario that matches, but list all of them if no argument is provided?

File: tools/psr/psrctl/cmd/start/start.go [hunk 3]
```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package start
+
```
Comment (mbenmclean) [inline tools/psr/psrctl/cmd/start/start.go:5]: I prefer run as the command name honestly. Any reason why you changed it?

File: tools/psr/psrctl/cmd/explain/explain.go (hunk 2 above)
Comment (pfmackin) [reply tools/psr/psrctl/cmd/explain/explain.go:35]: All for now, but a single one is coming in future PR
Explain lines up with kubectl explain.  List will show running scenarios.

File: tools/psr/psrctl/cmd/start/start.go (hunk 3 above)
Comment (pfmackin) [reply tools/psr/psrctl/cmd/start/start.go:5]: start / stop are symmetric and i think it is more accurate.  
Run is more of a steady state description (rstart a car then it is running, pod is running, startup probe, etc)

Comment (stevshap) [reply tools/psr/psrctl/cmd/start/start.go:5]: I had it as run orginally

File: tools/psr/psrctl/cmd/explain/explain.go (hunk 1 above)
Comment (pfmackin) [reply tools/psr/psrctl/cmd/explain/explain.go]: fixed

File: tools/psr/psrctl/cmd/explain/explain.go (hunk 2 above)
Comment (pfmackin) [reply tools/psr/psrctl/cmd/explain/explain.go:35]: i fixed the command help

**Review by stevshap — APPROVED** [review]

**Pull request merged**